br/backup-and-restore-storages.md

```diff
@@ -1,247 +1,229 @@
 ---
```

Comment:
after removing the external storage doc, what about dumping and lightning?

Reply:
Have removed description about dumpling and lightning from this file.